File: final-evaluation-generated-code/test-3/gemini.js
```javascript
async function subscribeToUpdateEvent() {
  const url = 'http://localhost:3000/http-express-calculator-simple/events/update';
  const eventSource = new EventSource(url);

  eventSource.onmessage = (event) => {
    const data = JSON.parse(event.data);
    console.log('Received update event:', data);
  };

  eventSource.onerror = (error) => {
    console.error('EventSource failed:', error);
    eventSource.close();
  };

  return () => {
    eventSource.close();
    console.log('Unsubscribed from update event');
  }
}


async function main() {
  const unsubscribe = await subscribeToUpdateEvent();
  // ... your code to process events ...
  setTimeout(() => unsubscribe(), 10000); // Unsubscribe after 10 seconds

}

main();
```

File: final-evaluation-generated-code/test-9/gemini.py
```python
import requests
import json
import sseclient

# SSE connection details
url = "http://localhost:3000/http-express-calculator-simple/events/update"
headers = {'Accept': 'text/event-stream'}

# Establish the SSE connection
response = requests.get(url, headers=headers, stream=True)
client = sseclient.SSEClient(response)

# Subscribe to events
for event in client.events():
    if event.event == 'message':
        print(f"Received event: {event.data}")
    elif event.event == "error":
        print(f"Error received: {event.data}")
        break

# Unsubscribe (Close the connection)
response.close()

```